src/openloco/input/mouse_input.cpp

```diff
@@ -307,6 +310,110 @@ namespace openloco::input
         }
     }
 
+    // 0x004C72ED
+    static void loc_4C72ED()
+    {
+        auto window = WindowManager::find(_pressedWindowType, _pressedWindowNumber);
+        if (window == nullptr)
+            return;
+
+        constexpr uint16_t horizontalFlags = scroll_flags::HSCROLLBAR_THUMB_PRESSED | scroll_flags::HSCROLLBAR_LEFT_PRESSED | scroll_flags::HSCROLLBAR_RIGHT_PRESSED;
+        constexpr uint16_t verticalFlags = scroll_flags::VSCROLLBAR_THUMB_PRESSED | scroll_flags::VSCROLLBAR_UP_PRESSED | scroll_flags::VSCROLLBAR_DOWN_PRESSED;
+
+        window->scroll_areas[_currentScrollOffset / sizeof(ui::scroll_area_t)].flags &= ~(verticalFlags | horizontalFlags);
+        WindowManager::invalidateWidget(_pressedWindowType, _pressedWindowNumber, _pressedWidgetIndex);
+    }
+
+    // 0x004C71F6
+    static void state_scroll_left(mouse_button cx, widget_index edx, ui::window* window, ui::widget_t* widget, int16_t x, int16_t y)
+    {
+        switch (cx)
+        {
+            case mouse_button::released:
+            {
+                if (edx != _pressedWidgetIndex)
+                {
+                    loc_4C72ED();
+                    return;
+                }
+                if (window->type != *_pressedWindowType || window->number != _pressedWindowNumber)
+                {
+                    loc_4C72ED();
+                    return;
+                }
+
+                if (_currentScrollArea == scrollview::scroll_part::hscrollbar_thumb)
+                {
+                    int16_t deltaX = x - _tooltipCursorX;
+                    _tooltipCursorX = x;
+                    scrollview::scroll_5_follow(_pressedWindowType, _pressedWindowNumber, _pressedWidgetIndex, _currentScrollOffset / sizeof(ui::scroll_area_t), deltaX);
```

Comment:
Again, I'm not following the `_currentScrollOffset / sizeof(ui::scroll_area_t)`.